lib/puppet/pops/loader/task_instantiator.rb

```diff
@@ -92,7 +92,9 @@ def self.create_task_type_from_hash(loader, typed_name, task_source, hash)
       parent_type = 'GenericTask'
     end
 
-    constants['executable'] = Pathname(task_source).relative_path_from(Pathname(loader.path) + 'tasks').to_s
+    if loader.path.nil?
+      return nil
```

Comment:
The line that sets constants['executable'] is brought back later - this was a mistake here.